e2e_tests/tests/fixtures/no_op/single-in-epochs.yaml

```diff
@@ -0,0 +1,26 @@
+description: noop_single
+checkpoint_storage:
+  type: shared_fs
+  host_path: /tmp
+  storage_path: determined-integration-checkpoints
+  save_trial_best: 30
+hyperparameters:
+  global_batch_size: 32
+  metrics_progression: decreasing
+  metrics_base: 0.9
+  metrics_sigma: 0
+searcher:
+  metric: validation_error
+  smaller_is_better: true
+  name: single
+  max_length:
+    epochs: 2
+reproducibility:
+  experiment_seed: 999
+records_per_epoch: 8000
```

Comment:
any particular reason why these need to be so long?

Reply:
`(2 * 8000 records) / (32 records/batch * 100 batches/step)` puts this at 5 steps, which I think is pretty reasonable.